### PR TITLE
License and license headers

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#!/usr/bin/env python3
```
Comment (goern) [inline setup.py]: two headers at once?!

File: thoth/lab/__init__.py [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#!/usr/bin/env python3
```
Comment (goern) [inline thoth/lab/__init__.py]: twice two headers?!

File: thoth/lab/graph.py [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#!/usr/bin/env python3
```
Comment (goern) [inline thoth/lab/graph.py]: ?! #wtf

File: thoth/lab/utils.py [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#!/usr/bin/env python3
```
Comment (goern) [inline thoth/lab/utils.py]: :)

File: thoth/lab/graph.py [hunk 5]
```diff
@@ -1,3 +1,21 @@
...
+# -*- coding: utf-8 -*-
+# thoth-naming-service
```
Comment (fridex) [inline thoth/lab/graph.py]: A leftover :(

**Pull request merged**